### ISSUE TITLE
Continous updates from onRangeChanged

### ISSUE THREAD

**Issue by Querschlag** [bendikv/simple-range-view#5]
It would be cool to have an additional property on the range view to make ```OnChangeRangeListener``` calls continuous and thus firing while moving the range line not just after the dragging gesture finished.

**Comment by bendikv** [comment]
Hello @Querschlag! You can use `rangeView.setOnTrackRangeListener()` for that.

**Comment by Querschlag** [comment]
You are right. Although I can only implement ```onStartRangeChanged``` or ```onEndRangeChanged```, otherwise I will end up with two calls. In my case (just moving the range) this would not be optimal.
